<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinPaperCatalog</class>
 <widget class="QWidget" name="WinPaperCatalog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WinArticleCatalog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_back">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
         <property name="icon">
          <iconset resource="MainWin.qrc">
           <normaloff>:/Icon/Resources/图标/back_1152px_1143771_easyicon.net.png</normaloff>:/Icon/Resources/图标/back_1152px_1143771_easyicon.net.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>38</width>
           <height>40</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>178</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btn_selectAll">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>全选/不选</string>
       </property>
       <property name="icon">
        <iconset resource="MainWin.qrc">
         <normaloff>:/Icon/Resources/图标/accept_page_128px_500987_easyicon.net.png</normaloff>:/Icon/Resources/图标/accept_page_128px_500987_easyicon.net.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_refresh">
       <property name="text">
        <string>重置显示</string>
       </property>
       <property name="icon">
        <iconset resource="MainWin.qrc">
         <normaloff>:/Icon/Resources/图标/matte_white_square_icon_arrows_arrow_circle_refresh_512px_570726_easyicon.net.png</normaloff>:/Icon/Resources/图标/matte_white_square_icon_arrows_arrow_circle_refresh_512px_570726_easyicon.net.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_search">
       <property name="text">
        <string>高级搜索</string>
       </property>
       <property name="icon">
        <iconset resource="MainWin.qrc">
         <normaloff>:/Icon/Resources/图标/search_black_and_white.png</normaloff>:/Icon/Resources/图标/search_black_and_white.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_print">
       <property name="text">
        <string>报表打印</string>
       </property>
       <property name="icon">
        <iconset resource="MainWin.qrc">
         <normaloff>:/Icon/Resources/图标/printer_128px_523218_easyicon.net.png</normaloff>:/Icon/Resources/图标/printer_128px_523218_easyicon.net.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_articleInfo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>66</height>
        </size>
       </property>
       <property name="text">
        <string>文章详情</string>
       </property>
       <property name="icon">
        <iconset resource="MainWin.qrc">
         <normaloff>:/Icon/Resources/图标/book_128px_501187_easyicon.net.png</normaloff>:/Icon/Resources/图标/book_128px_501187_easyicon.net.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_delete">
       <property name="text">
        <string>删除文章</string>
       </property>
       <property name="icon">
        <iconset resource="MainWin.qrc">
         <normaloff>:/Icon/Resources/图标/delete_page_128px_501003_easyicon.net.png</normaloff>:/Icon/Resources/图标/delete_page_128px_501003_easyicon.net.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_add">
       <property name="text">
        <string>增加文章</string>
       </property>
       <property name="icon">
        <iconset resource="MainWin.qrc">
         <normaloff>:/Icon/Resources/图标/add_page_128px_500989_easyicon.net.png</normaloff>:/Icon/Resources/图标/add_page_128px_500989_easyicon.net.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="m_paperTable">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWin.qrc"/>
 </resources>
 <connections/>
</ui>
